### PR TITLE
Add more help on how to deploy to Shiny

### PR DESCRIPTION
**Pull request by lvarin**

### CODE REVIEW

**Comment by rajanssi** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -63,4 +63,14 @@ You might also want to delete the persistent volume created by the setup by
 
 ## Publish an application to Shiny
 
-Use the terminal in the Rstudio interface to copy your application to /srv/shiny-server/
+Shiny is an application server for R. This template runs it by default, but if you develop an application and what to deploy it to Shiny, you will need to do a manual copy of the files. The procedure is:
```
Comment (jemaltahir) [inline README.md]:
```suggestion
Shiny is an application server for R. This template runs it by default, but if you develop an application and want to deploy it to Shiny, you will need to do a manual copy of the files. The procedure is:
```

**Review by jemaltahir — APPROVED** [review]
fixed typo

**Review by jemaltahir — APPROVED** [review]

**Review by jemaltahir — COMMENTED** [review]
LGTM

**Comment by jemaltahir** [comment]
LGTM

**Pull request merged**